index.js

```diff
@@ -18,6 +37,96 @@ inputElement.addEventListener('input', event => {
       console.error(error);
       outputInstructions.style.display = '';
     });
+}
+
+inputElement.addEventListener('paste', event => {
```

Comment:
The function inside this call should be factored out so that unit tests can work with it.